### ISSUE TITLE
Better favicon

### ISSUE THREAD

**Issue by mpsq** [mpsq/arewewaylandyet#22]

**Comment by Bounty780** [comment]
This might be an interesting feat. Where would you like to get it?

**Comment by DestroyerXyz** [comment]
![wayland_favicon](https://user-images.githubusercontent.com/70191184/99870638-c614c380-2bfa-11eb-9616-951bf7131f86.png)

How is this? I think it's not that great.

**Comment by mpsq** [comment]
Hi Shreyans,

Thanks for your contribution! I really like your idea, I think this can be a good base. A few things come to my mind, the first is how much details your version has, I think the tiny time markers won't be that "nice" once the icon is small. I am thinking of a clock like [this one](https://boxicons.com/) (search for "time") might work nicely?

Also, we can probably move away from the official Wayland icon, we can use a generic "w" (rendered in an open source sans-serif font) on a yellow background (this time using the official yellow).

What do you think?

**Comment by DestroyerXyz** [comment]
I agree that this will not look good when it is small but I think we should still use the Wayland Logo.
